### PR TITLE
form added

### PR DESCRIPTION
**Pull request by aayushtailang1911**
i have added form in singup component

### CODE REVIEW

**Review by PruthvirajNitor — APPROVED** [review]
Good

File: amazon-clone/src/components/SignUp.vue [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<template></template>
+<template>
+    <form></form>
+</template>
```
Comment (abhijeet2902) [inline amazon-clone/src/components/SignUp.vue:3]: Insert some inputs

**Review by abhijeet2902 — CHANGES_REQUESTED** [review]
Please review your code

**Comment by abhijeet2902** [comment]
closing do you work and again raise a pr

**Pull request merged**